README.md

```diff
@@ -16,6 +16,9 @@ Try to use `require` for all your embedded resources, to process them with webpa
 ``` jade
 div
   img(src=require("./my/image.png"))
+
+//Use #{...} for inline-style images  
+div(style='background-image:url(#{require("./my/image.png")})')

```

Comment:
This syntax has been removed in Pug 2.0.0.

Now you need to use

```pug
div(style='background-image:url(' + require("./my/image.png") + ')')
//- or
div(style=`background-image:url(${require("./my/image.png")})`)

Reply:
require is not a function !!